src/js/actions.js

```diff
@@ -124,11 +123,7 @@ function submitToUrl(body, submitUrl, trackingPrefix) {
 
     req.setRequestHeader('X-Key-Inflection', 'camel');
     req.setRequestHeader('Content-Type', 'application/json');
-
```

Comment:
This is most likely just a leftover from the VA-to-USFS migration since USFS never had any concept of users.